### DISCUSSION TITLE
Add Newer Python Versions to Guake

### DISCUSSION THREAD

**Discussion by novaTopFlex** [Guake/guake#2246] [Ideas]
I have repeatedly attempted to change the regex to enable python3.10, python3.11, python3.12, ... to be indexed for Python as default shell. Right now the newest version of Python that can be indexed by the regex is the python3.9 series, and as an Ubuntu 22.04 LTS and Deadsnakes user I have the default python3.10 installed as well as python3.11, python3.12, and python3.13, yet my installation of python3.9 is the only installation that can be indexed successfully.

**Comment by Davidy22** [comment]
Linking #2242, which I presume you created to resolve part of this. The other half that I presume you're asking for is an update of the `setup.cfg` file, although I suspect I might not fully have a handle on what's going on here because if it was that file 3.11 should be working.
